### PR TITLE
Update slack_etiquette.md to include note on slack messages not being deletable.

### PR DESCRIPTION
**Pull request by berlin-pyladies**
As per the [conversation](https://pyladies.slack.com/archives/C1F9A0K7F/p1707313118839909?thread_ts=1707312870.081709&cid=C1F9A0K7F) in the organiser's slack channel, I (@sleepypioneer) on behalf of the PyLadies community am proposing this update :)

### CODE REVIEW

**Comment by sleepypioneer** [comment]
@Mariatta @willingc tagging you folks for review :)

File: process/slack_etiquette.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ Questions in #general tend to get answered quickly, or they get pushed off the s
 Conversely, if a channel is particularly quiet, please do not post "this channel is quiet".  Instead, try posting a question/article/or discussion point that others in the channel might comment on or enjoy.
 
 
+### :star2: Messages in Slack cannot be deleted
+
+We allow users to edit their messages for a short period of time after posting, ie to fix typos. But we keep messages as *proof*, for when people post inappropriate content.
```
Comment (willingc) [inline process/slack_etiquette.md:44]:
```suggestion
We allow users to edit their messages for a short period of time after posting, i.e. to fix typos. We store messages as *historical records*; so please refrain from posting inappropriate content.
```

**Review by willingc — COMMENTED** [review]
Overall, this looks good. I made a few suggestions on the addition. Thanks.

File: process/slack_etiquette.md (hunk 1 above)
Comment (sleepypioneer) [inline process/slack_etiquette.md:44]:
```suggestion
We allow users to edit their messages for a short period of time after posting, i.e. to fix typos. We store messages as *historical records*; so please refrain from posting inappropriate content.
```

File: process/slack_etiquette.md [hunk 2]
```diff
@@ -39,6 +39,10 @@ Questions in #general tend to get answered quickly, or they get pushed off the s
 Conversely, if a channel is particularly quiet, please do not post "this channel is quiet".  Instead, try posting a question/article/or discussion point that others in the channel might comment on or enjoy.
 
 
+### :star2: Messages in Slack cannot be deleted
+
+We allow users to edit their messages for a short period of time after posting, i.e. to fix typos. We store messages as *historical records*; so please refrain from posting inappropriate content.
```
Comment (sleepypioneer) [inline process/slack_etiquette.md:44]: @willingc thanks for the review, updated the text as per your review 🙏

**Review by willingc — APPROVED** [review]
Thanks for the addition. Happy to merge.

**Pull request merged**